fossunited/api/hackathon.py

```
@@ -612,6 +612,79 @@ def get_issue_pr_title(url: str) -> dict:
+    current_user = frappe.session.user
+    if current_user == "Guest":
+        frappe.throw("Authentication required", frappe.AuthenticationError)
+
+    allowed_statuses = ("Accepted", "Rejected")
+    if status not in allowed_statuses:
+        frappe.throw(
+            f"Invalid status '{status}'. Allowed values: {', '.join(allowed_statuses)}",
+            frappe.ValidationError,
+        )
+
+    request_doc = frappe.get_doc(JOIN_TEAM_REQUEST, request_id)
```

Ruff (0.15.2)

[error] Undefined name `JOIN_TEAM_REQUEST`

(F821)